toolkit/scripts/tools.mk

```diff
@@ -80,19 +76,14 @@ go_ldflags := 	-X github.com/microsoft/azurelinux/toolkit/tools/internal/exe.Too
 
 # Matching rules for the above targets
 # Tool specific pre-requisites are tracked via $(go-util): $(shell find...) dynamic variable defined above
-$(TOOL_BINS_DIR)/%: $(go_common_files) $(GO_LICENSES_TOOL)
+$(TOOL_BINS_DIR)/%: $(go_common_files)
+	$(MAKE) license-scan
```

Comment:
Why not set `license-scan` as a dependency instead of recursively calling `make`?